cypress/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "extends": "../tsconfig.json",
+  "compilerOptions": {
+    "strict": true,
+    "baseUrl": "../node_modules",
+    "target": "es5",
```

Comment:
Any reason this is not `"module": "esnext"`, which is what we have in the root tsconfig?